src/1_utils.js

```diff
@@ -439,4 +439,21 @@ utils.scrapeOpenGraphContent = function(property, content) {
 	return content;
 };
 
+/**
+ * Search for hosted deep link data on the page, as outlined here https://dev.branch.io/getting-started/hosted-deep-link-data/guide/#adding-metatags-to-your-site
+ */
+utils.scrapeHostedDeepLinkData = function() {
+	var params = {};
+
+	var metas = document.getElementsByTagName("meta");
+
+	for (var i = 0; i < metas.length; i++) {
+		if (metas[i].getAttribute("name").startsWith("branch:deeplink:")) {
```

Comment:
`startsWith` cannot be used:

http://kangax.github.io/compat-table/es6/#test-String.prototype.startsWith